config/initializers/constants.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Constants
+  HOST_URI = URI.parse(ENV.fetch('HOST'))
```

Comment:
@citizen428 think the env var should be named HOST_URI for consistency or do you like the distinction between HOST (the env var) and HOST_URI (the parsed constant)?